frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -672,6 +674,17 @@ export const dashboardLogic = kea<dashboardLogicType>([
                 return candidateShortest?.isValid() ? candidateShortest : null
             },
         ],
+        blockRefresh: [
+            (s) => [s.lastRefreshed],
+            (lastRefreshed) => {
+                return (
+                    !!lastRefreshed &&
+                    dayjs()
```

Comment:
we're importing `import { dayjs, now } from 'lib/dayjs'` we have a custom wrapper around dayjs (IIRC to control plugin init)

I think you have to use `now()` not `dayjs()` (but haven't checked)